CHANGELOG.md

```diff
@@ -1,8 +1,18 @@
-# dbt_microsoft_ads version.version
+# dbt_microsoft_ads v0.9.1
+
+## Feature Update
+
+- Made `ad_name` in `stg_microsoft_ads__ad_history` customizable via a new variable `microsoft_ads__ad_name_selector`. By default this is determined using `title_part_1`, but you override this by including the configuration shown below in your `dbt_project.yml` file. For more information, refer to the [README](https://github.com/fivetran/dbt_microsoft_ads_source/blob/main/README.md#change-how-ad-name-is-determined).
+
+```yml
+vars:
+    microsoft_ads__ad_name_selector: coalesce(title_part_2, title_part_1) # using `title_part_2`, with `title_part_1` as a fallback if the former is `null`.
+```
 
 ## Documentation
 - Added Quickstart model counts to README. ([#39](https://github.com/fivetran/dbt_microsoft_ads/pull/39))
 - Corrected references to connectors and connections in the README. ([#39](https://github.com/fivetran/dbt_microsoft_ads/pull/39))
+- Updated License
```

Comment:
needs PR link

Reply:
I did forget to add the PR above! Added that in and that should cover this